modules/search/instant-search/components/overlay.scss

```diff
@@ -0,0 +1,21 @@
+.jetpack-instant-search__overlay {
```

Comment:
We'll probably need a ridiculous value for `z-index` to ensure the overlay is the top most layer on any site. I'd recommend starting with something like `9999999999999`. :)

Reply:
Agreed :)